src/hdf5/HDF5IO.cpp

```diff
@@ -48,15 +50,16 @@ Status HDF5IO::open(bool newfile)
   if (opened)
     return Status::Failure;
 
-  FileAccPropList props = FileAccPropList::DEFAULT;
+  FileAccPropList fapl = FileAccPropList::DEFAULT;
+  H5Pset_libver_bounds(fapl.getId(), H5F_LIBVER_LATEST, H5F_LIBVER_LATEST);
 
   if (newfile)
     accFlags = H5F_ACC_TRUNC;
   else
     accFlags = H5F_ACC_RDWR;
 
   file = std::make_unique<H5::H5File>(
-      getFileName(), accFlags, FileCreatPropList::DEFAULT, props);
+      getFileName(), accFlags, FileCreatPropList::DEFAULT, fapl);
```

Comment:
In line 70 in `HDF5IO::close()`, it looks like the function doesn't actually close the HDF5 file. I think it should call:

```
    if(this->file != nullptr && opened)
    {
       H5Fclose(this->file->getId());
    }
``

Reply:
updated using the `file->close()` function, I think they are equivalent.